plugins/cdi/cdi.plugin.zsh

```diff
@@ -0,0 +1,20 @@
+# vim: set sw=4 expandtab:
+#
+# Licence: GPL
+# Created: 2016-09-27 14:41:53+02:00
+# Main authors:
+#     - Jérôme Pouiller <jezz@sysmic.org>
+#
+# "Change Directory Interactive"
+#
+# Allow to change directory by editing current working directory path.
+#
```

Comment:
Delete this whole comment and put the relevant information in a README (for example, [the `pj` plugin](https://github.com/robbyrussell/oh-my-zsh/blob/master/plugins/pj/README.md)).